src/bin/mir2wasm.rs

```diff
@@ -1,5 +1,4 @@
-#![feature(rustc_private, custom_attribute, link_args)]
-#![allow(unused_attributes)]
```

Comment:
yeah I think it was just a leftover from the miri bin and AFAICT we're not going to use miri with those attributes anyway :)